app/models/legislator.rb

```diff
@@ -132,10 +132,14 @@ def district_code
     district.district if district
   end
 
+  def image_url(size:'225x275')
```

Comment:
Unused method argument - `size`. You can also write as `image_url(*)` if you want the method to accept any arguments but don't care about them.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.